# start_exploration.srv
bool success 
# Request
---
bool success 


